src/utils/components/ConfirmDelete.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import '../styles/ConfirmDelete.css';
+import PropTypes from 'prop-types';
+import api from '../../services/api'
+
+const ConfirmDelete = (props) => {
+
+  const [value,setValue] = useState();
```

Comment:
Esse nome de variável não deixa claro do que se trata. Recomendo trocar por algo que indique se trata de do nome de um laboratório, como `labName`, `laboratoryName` ou mesmo `name`, por exemplo.

```suggestion
  const [labName, setLabName] = useState();
```